### ISSUE TITLE
REST API: IP address in path *and* get parameter

### ISSUE THREAD

**Issue by ghost** [aaronkaplan/torexitnodes_simple#6]
Example URL for the search when clicking on the "Search" button:
http://127.0.0.1:5000/ip/35.0.127.5?ip=35.0.127.5
Why the IP address two times? The latter seems to be ignored.

**Comment by aaronkaplan** [comment]
> On 08.08.2019, at 16:18, Wagner <notifications@github.com> wrote:
> 
> Example URL for the search when clicking on the "Seach" button:
> http://127.0.0.1:5000/ip/35.0.127.5?ip=35.0.127.5
> Why the IP address two times? The latter seems to be ignored.

Yup, nice catch. Any better idea on how to make this possible via the JS search onSubmit() code?
I saw this as well but did not find any better idea so far (since I am not a JS coder)
